weathervane.config.docker.custom

```diff
@@ -0,0 +1,119 @@
+{
+
+  "dockerNamespace" : "yourRepository",
+  "dockerHosts" : [
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionWeb1", "dockerPort" : 2376, "dockerNet" : "bridge" }, 
+    { "name" : "AuctionBid1", "dockerPort" : 2375, "dockerNet" : "bridge" }, 
+    { "name" : "AuctionBid2", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionBid3", "dockerPort" : 2376, "dockerNet" : "host" }, 
```

Comment:
1 "bridge" and 2 "host" for AuctionBids?  Is that intended?

Reply:
Yes.  This config file is just intended to show the variety of things you can specify.  I wouldn't expect anyone to use it this way.